experimental/PropertyDDS/examples/property-inspector/README.md

```diff
@@ -1,8 +1,24 @@
 # @fluid-experimental/property-inspector
```

Comment:
I meant to clean this up a bit (and properly run the generator to make sure this works) before pushing it.
I'll make some adjustments.